tests/Integration/Auth/Fixtures/AuthenticationTestUser.php

```diff
@@ -12,14 +12,14 @@ class AuthenticationTestUser extends Authenticatable
     /**
      * The attributes that aren't mass assignable.
      *
-     * @var array
+     * @var string[]
      */
     protected $guarded = [];
 
     /**
      * The attributes that should be hidden for arrays.
      *
-     * @var array
+     * @var string[]
```

Comment:
wrong. `string[]` is an alias for `array<int,string>` but this type is `array<string,string>`.

Reply:
I agree that some of my fixes are wrong, but this one doesn't look like wrong.

```php
protected $hidden = [
        'password', 'remember_token',
];
```
Is equivalent for:
```php
protected $hidden = [
        0 => 'password', 1 => 'remember_token',
];
```